ui/imports/shared/stores/send/TransactionStore.qml

```diff
@@ -226,4 +227,12 @@ QtObject {
     function formatCurrencyAmountFromBigInt(balance, symbol, decimals, options = null) {
         return currencyStore.formatCurrencyAmountFromBigInt(balance, symbol, decimals, options)
     }
+
+    function updateRecentRecipientsActivity(walletAccount) {
```

Comment:
probably it would be cleaner to take just an address here

Reply:
Initially I agreed but after looking at it, not sure.. then in all the callers, I need to previously check if `walletAccount` is an object, so better to to the check only in one place?

Reply:
Ok, let's leave as it is, maybe later at some point we will take a closer look.

But now this function logic is quite strange. Caller doesn't know is something was set or not without doing similar inspection on it's own side.